src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -98,6 +100,8 @@
         this.samlRolesSeparator = settings.get("roles_seperator");
```

Comment:
Convert to local variable, there is no need to keep it as the state of `AuthTokenProcessorHandler`.

Reply:
Done.